trikot-viewmodels-declarative-flow/viewmodels-declarative-flow/src/commonMain/kotlin/com/mirego/trikot/viewmodels/declarative/components/factory/VMDComponents.kt

```diff
@@ -136,13 +161,31 @@ object VMDComponents {
                 VMDButtonViewModelImpl(coroutineScope, textPair)
                     .apply(closure)
 
+            fun withTextPair(
+                textPairFlow: VMDTextPairFlowContent,
+                coroutineScope: CoroutineScope,
+                closure: VMDButtonViewModelImpl<VMDTextPairContent>.() -> Unit = {}
```

Comment:
This is a bit specific to your use case and I feel this shouldn't go in the public API.

Reply:
I actually did not need it for my project, but I figured that if we were to make it compatible with only VMDTextComponent, might as well also do it for every component using texts